tpm/tss-minimal/esys-sys/build.rs

```diff
@@ -0,0 +1,57 @@
+fn main() {
+    let lib_cfg = pkg_config::Config::new()
```

Comment:
For building, yes. tpm2-tss already has the required `.pc` files. A dev package is required anyway for `bindgen`.

`bindgen` is used because writing the bindings by hand would be painful (but not impossible) due to `tss2_tpm2_types.h`. I am open to taking the bindgen output and dumping it into `*-sys` directly given that vendoring is going to be a challenge in and of itself.

Reply:
I'm not saying to take bindgen out, just that either we have to make corresponding devel packages, or else we have to add something to work as an alternative, like an env var that bypasses pkg-config for our own builds.

For example, you've added tpm2-tss-devel as a compile-time dep for CentOS 7, but this is an old version and we want to link to our own vendored one, yes?

Reply:
As discussed offline, there will be an option to vendor regardless of OS package availability.